testing/test_pytester.py

```diff
@@ -394,3 +394,8 @@ def test_preserve_container(self, monkeypatch, path_type):
         assert getattr(sys, path_type) == original_data
         assert getattr(sys, other_path_type) is original_other
         assert getattr(sys, other_path_type) == original_other_data
+
+
+def test_testdir_subprocess(testdir):
+    testfile = testdir.makepyfile("def test_one(): pass")
+    assert testdir.runpytest_subprocess(testfile).ret == 0
```

Comment:
this is only a proper regression test when invoked in a virtualenv with `enum34` installed under python3.5+

since this is pretty obscure, I don't think it's worth complicating our test suite to accommodate that.

either way though, this test still serves a purpose: as a characterization test for `runpytest_subprocess` which wasn't called elsewhere

Before this, it failed as follows:

```
___________________________ test_testdir_subprocess ____________________________

testdir = <Testdir local('/tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0')>

    def test_testdir_subprocess(testdir):
        testfile = testdir.makepyfile("def test_one(): pass")
>       assert testdir.runpytest_subprocess(testfile).ret == 0
E       AssertionError: assert 1 == 0
E        +  where 1 = <_pytest.pytester.RunResult object at 0x7f5e663a9c18>.ret
E        +    where <_pytest.pytester.RunResult object at 0x7f5e663a9c18> = <bound method Testdir.runpytest_subprocess of <Testdir local('/tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0')>>(local('/tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0/test_testdir_subprocess.py'))
E        +      where <bound method Testdir.runpytest_subprocess of <Testdir local('/tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0')>> = <Testdir local('/tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0')>.runpytest_subprocess

/tmp/pytest/testing/test_pytester.py:401: AssertionError
----------------------------- Captured stdout call -----------------------------
running: /tmp/pytest/venv/bin/python3 /tmp/pytest/venv/lib/python3.6/site-packages/pytest.py --basetemp=/tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0/runpytest-0 /tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0/test_testdir_subprocess.py
     in: /tmp/pytest-of-asottile/pytest-29/test_testdir_subprocess0
----------------------------- Captured stderr call -----------------------------
Traceback (most recent call last):
  File "/tmp/pytest/venv/lib/python3.6/site-packages/pytest.py", line 9, in <module>
    from _pytest.config import main, UsageError, cmdline, hookspec, hookimpl
  File "/tmp/pytest/venv/lib/python3.6/site-packages/_pytest/config/__init__.py", line 3, in <module>
    import argparse
  File "/usr/lib/python3.6/argparse.py", line 86, in <module>
    import collections as _collections
  File "/tmp/pytest/venv/lib/python3.6/collections/__init__.py", line 32, in <module>
    from reprlib import recursive_repr as _recursive_repr
  File "/tmp/pytest/venv/lib/python3.6/site-packages/reprlib.py", line 3, in <module>
    from repr import *
ModuleNotFoundError: No module named 'repr'
```